packages/web-components/src/components/masthead/masthead-cart.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+import { html, LitElement } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import settings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import { carbonElement as customElement } from '@carbon/web-components/es/globals/decorators/carbon-element';
+import ShoppingCart20 from '@carbon/web-components/es/icons/shopping--cart/20.js';
+import styles from './masthead.scss';
+import LocaleAPI from '@carbon/ibmdotcom-services/es/services/Locale/Locale.js';
+import SAPCommerceAPI from '@carbon/ibmdotcom-services/es/services/SAPCommerce/SAPCommerce.js';
+
+const { prefix, stablePrefix: c4dPrefix } = settings;
+
+/**
+ * The Cart icon in the masthead.
+ *
+ * @element c4d-masthead-cart
+ * @csspart cart-link - The masthead cart link. Usage: `c4d-masthead-cart::part(cart-link)`
+ */
+@customElement(`${c4dPrefix}-masthead-cart`)
+class C4DMastheadCart extends StableSelectorMixin(LitElement) {
+  /**
+   * The `aria-label` attribute for the link.
+   */
+  @property({ attribute: 'link-label' })
+  linkLabel = 'Cart';
+
+  /**
+   * Tracks whether the user has an active cart to control the display.
+   */
+  @property({ attribute: 'has-active-cart', reflect: true, type: Boolean })
+  hasActiveCart = false;
```

Comment:
Should this be something configurable by an author? Perhaps we should use state and toggle a class on the rendered link?

Reply:
Good call. I've refactored based on your suggestion to just toggle a `hidden` attribute when we shouldn't be showing the cart. It simplifies the render method as well as the Sass. Let me know what you think.